tutorials/08-voting/contracts/VotingTutorialAdministration.cdc

```diff
@@ -0,0 +1,139 @@
+/*
+*
+*   In this example, we want to create a simple voting contract
+*   where a polling place issues ballots to addresses.
+*
+*   The run a vote, the Admin deploys the smart contract,
+*   then initializes the proposals.
+*   The array of proposals cannot be modified after it has been initialized.
+*
+*   Users can create ballots and vote only with their VotingTutorialGovernanceToken balance prior to when
+*   proposal was created.
+*
+*   Every user with a ballot is allowed to approve their chosen proposals.
+*   A user can choose their votes and cast them
+*   with the tx_05_SelectAndCastVotes.cdc transaction.
+*
+*/
+
+import VotingTutorialGovernanceToken from "./VotingTutorialGovernanceToken.cdc"
+
+pub contract VotingTutorialAdministration {
+
+    // dictionary of proposals to be approved
```

Comment:
Sorry for nitpicks - It is always beautiful to start with the capital letter as the code comment statement